src/canopy_server/app.py

```diff
@@ -288,13 +313,8 @@ async def startup():
 
 
 def _init_routes(app):
-    # Include the application level router (health, shutdown, ...)
-    app.include_router(application_router, include_in_schema=False)
```

Comment:
We need to remember to say in the next release's changelog that routes without `/v1` aren't supported anymore.

Reply:
I will write in PR description not to forget.